frontend-nextjs/README.md

```
@@ -0,0 +1,104 @@
+Backend http://localhost:8000/
+Streamlit http://localhost:8501/
+CTRL+Cで停止
+```
+
+## devcontainer設定
+`.devcontainer/frontend-nextjs/devcontainer.json`　ファイルを作成
+  - `docker-compose.yml`をベースに `VS Code` の `devcontainer` を設定
+  - コンテナ内に拡張機能として`es7-react`と`biome`を導入
+  - 起動時に`pnpm install`を実行して、ライブラリを再導入
+
+- VSCode拡張機能の設定
```

Markdownlint

Expected: 2; Actual: 0
Inconsistent indentation for list items at the same level

(MD005, list-indent)